backend/tests/unit/dal/Services/Admin/ProjectRiskServiceTest.cs

```diff
@@ -0,0 +1,191 @@
+using FluentAssertions;
+using Microsoft.EntityFrameworkCore;
+using Pims.Core.Comparers;
+using Moq;
+using Pims.Core.Extensions;
+using Pims.Core.Test;
+using Microsoft.Extensions.Options;
+using Pims.Dal.Entities.Models;
+using Pims.Dal.Entities;
+using Pims.Dal.Exceptions;
+using Pims.Dal.Security;
+using Pims.Dal.Services.Admin;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using Xunit;
+using Entity = Pims.Dal.Entities;
+
+namespace Pims.Dal.Test.Services.Admin
+{
+    [Trait("category", "unit")]
+    [Trait("category", "dal")]
```

Comment:
It seems like a lot of the other /Admin tests have Trait("admin", "area"). Confirm that you don't need that here and in the next text.

Reply:
Good catch, thank you